docs/mp/guides/38_testing_junit5.adoc

```diff
@@ -0,0 +1,271 @@
+///////////////////////////////////////////////////////////////////////////////
+
+    Copyright (c) 2021 Oracle and/or its affiliates.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+///////////////////////////////////////////////////////////////////////////////
+
+= Helidon MP Testing with JUnit 5 Guide
+:h1Prefix: MP
+:description: Helidon testing with JUnit 5
+:keywords: helidon, testing, microprofile, guide, JUnit 
+:common-page-prefix-inc: ../../shared/common_prereqs/common_prereqs.adoc
+
+This guide describes how to write and execute tests for your MicroProfile applications in a JUnit 5 environment using optimized customizations.
+
+== What You Need
+
+For this 20 minute tutorial, you will need the following:
+include::{common-page-prefix-inc}[tag=common-prereqs]
+
+
+== Dependencies
+To start using this feature, add the following dependencies to the testing module:
+
+[source,xml]
+.Maven dependencies
+----
+<dependencies>
```

Comment:
@Tomas-Kraus I added the additional dependency for junit jupiter but I'm not sure if I used the correct version (5.5.2) or formatting.

Reply:
The version does not need to be defined (when using our application parent poms - it is managed by Helidon)

Reply:
removed version from dependencies